polkadot/node/core/pvf/src/prepare/worker_intf.rs

```diff
@@ -157,7 +150,8 @@ pub async fn start_work(
 						prepare_result,
```

Comment:
nit: for a bit more clarity:

```suggestion
						prepare_worker_result,
```